Meta/Lagom/Tools/CodeGenerators/LibTimeZone/GenerateTimeZoneData.cpp

```diff
@@ -316,8 +316,7 @@ static ErrorOr<void> parse_time_zones(StringView time_zone_path, TimeZoneData& t
     Vector<TimeZoneOffset>* last_parsed_zone = nullptr;
 
     while (TRY(file->can_read_line())) {
-        auto nread = TRY(file->read_line(buffer));
-        StringView line { buffer.data(), nread };
+        StringView line = TRY(file->read_line(buffer));
```

Comment:
`auto`? just caught my eye because this (and line 376) are the only place in this commit where `StringView` is used explicitly.

Reply:
:+1: